### DISCUSSION TITLE
Deleting the content of a table

### DISCUSSION THREAD

**Discussion by OriBennett** [noam-honig/radweb#341] [Q&A] [answered]
I'm trying to delete the full content of a table for seeding purposes. Is there a way to do it without getting all the contents then iterating over them?

**Comment by noam-honig** [comment, marked as answer]
Hi @OriBennett, although there is no built-in way of doing that, it is very easy to implement yourself.

Checkout the following article on how you can perform database operations directly:
https://remult.dev/docs/running-sql-on-the-server.html#using-the-underlying-database

In short, if it's an sql database, here's the code:
```ts
SqlDatabase.getDb().execute(`truncate table ${repo(City).metadata.dbName}`)
```
